tests/baselines/reference/exportEqualsAmd.js

```diff
@@ -4,6 +4,6 @@ export = { ["hi"]: "there" };
 //// [exportEqualsAmd.js]
 define(["require", "exports"], function (require, exports) {
     "use strict";
-    return (_a = {}, _a["hi"] = "there", _a);
+    return _a = {}, _a["hi"] = "there", _a;

```

Comment:
Is the removal of the parenthesis in this test output and the one below intentional?